### PR TITLE
Blog updating

### PR DESCRIPTION
**Pull request by LouiseMcMahon**

### CODE REVIEW

File: _posts/2014-02-20-february-the-15th-build-day.md [hunk 1]
```diff
@@ -37,14 +37,15 @@ After successfully removing an amount of burnt plastic from the extruder. We did
...
 
 **Arduino Robots**
 
 We also had a group member bring in an Arduino robot an AAR-04. It was programmed from the Arduino IDLE. There are two motors that drive the device back and forwards and a ball bearing at the front that keeps it in line. There are also some sensors at the front of the board, just by the ball bearing, these allow the robot to stop before it hit a surface.
 
 You can take a look at the video below of the robot working…
 
-<iframe allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="" frameborder="0" height="329" loading="lazy" src="https://www.youtube.com/embed/7jq4e9WCAcA?feature=oembed" title="Arduino AAR-04 at #shhm" width="584"></iframe>
\ No newline at end of file
+<iframe allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="" frameborder="0" height="329" loading="lazy" src="https://www.youtube.com/embed/7jq4e9WCAcA?feature=oembed" title="Arduino AAR-04 at #shhm" width="584"></iframe>
```
Comment (LouiseMcMahon) [inline _posts/2014-02-20-february-the-15th-build-day.md:50]: Ill add a layout for youtube so they can be included without HTML

File: _posts/2013-03-29-latest-meet-up.md [hunk 2]
```diff
@@ -22,8 +22,9 @@ This week on the 25th of march was our third meet up of the year. This month a n
 One of the first things that took place was the set up of the mendel 90 and the installation on the software. A computer and place
```
Comment (LouiseMcMahon) [inline _posts/2013-03-29-latest-meet-up.md:22]: Can we remove the span above this comment

File: _posts/2013-11-27-november-the-25th-meet-up.md [hunk 3]
```diff
@@ -52,4 +52,5 @@ We only recently found out about a company called ‘We Do 3D Printing’ who ar
 **Next Build Day**
 
 We now have a build day planned for the 14th of December between 10am to 4pm. You can bring along your projects, get advice from other group members and have access to the Refab Lab in Access Space with the Laser Cutter, CNC Router, 3D Printers and more.  
-Let us know if you are coming ether by our <a>forum</a> or on Twitter [@SHHMakers](http://twitter.com/shhmakers).
\ No newline at end of file
+Let us know if you are coming ether by our <a>forum</a> or on Twitter [@SHHMakers](http://twitter.com/shhmakers).
```
Comment (LouiseMcMahon) [inline _posts/2013-11-27-november-the-25th-meet-up.md:55]: Remove the <a>

**Review by LouiseMcMahon — COMMENTED** [review]
Added some comments

**Review by thermalhound — CHANGES_REQUESTED** [review]
The leftover html tags have been removed

File: _posts/2014-02-20-february-the-15th-build-day.md (hunk 1 above)
Comment (thermalhound) [reply _posts/2014-02-20-february-the-15th-build-day.md:50]: Thanks Louise, that'll be a great help as quite a lot of the later posts have embedded youtube videos

Comment (LouiseMcMahon) [reply _posts/2014-02-20-february-the-15th-build-day.md:50]: Thats in main now

**Pull request merged**